www/assets/app.css

```diff
@@ -22,8 +22,47 @@ h1, h4 {
 h1 {
   border-bottom: 2px solid #222;
   text-align: left;
+  line-height: 1.5;
 }
 
+button {
+  border-radius: 5px;
```

Comment:
We could make a '.btn' class here and a '.btn-small' modifier, might be nicer than styling the core elements.